### PR TITLE
[UIE-162] Update template to include link to Storybook.

### PR DESCRIPTION
**Pull request by cahrens**
### Jira Ticket: https://broadworkbench.atlassian.net/browse/UIE-162

Updated PR template to include a link to our Storybook library.

### CODE REVIEW

**Review by nawatts — APPROVED** [review]

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -7,13 +7,16 @@
...
-- 
+-
 
 ### Why
-- 
+-
 
 ### Testing strategy
+<!-- Note that changes impacting components in Storybook stories can be viewed at https://www.chromatic.com/library?appId=65fc89c9335768720ff8605a.
+Choose the branch corresponding to this PR, and changes can be reviewed by commit. --->
```
Comment (msilva-broad) [inline .github/pull_request_template.md:17]: this is a great step... maybe at a later point we can look at a way to have the correct branch already baked into the link?

**Review by msilva-broad — APPROVED** [review]
exciting stuff!

File: .github/pull_request_template.md (hunk 1 above)
Comment (cahrens) [reply .github/pull_request_template.md:17]: I don't think there is a way-- when you select a branch, what gets added to the URL as a parameter is the Chromatic build number (simply an increasing integer).

Comment (cahrens) [reply .github/pull_request_template.md:17]: Ooooh, I was wrong! `https://<branch>--65fc89c9335768720ff8605a.chromatic.com`

Comment (nawatts) [reply .github/pull_request_template.md:17]: It looks like the Chromatic action outputs the build URL / preview URL.

https://www.chromatic.com/docs/github-actions/#outputs

Comment (cahrens) [reply .github/pull_request_template.md:17]: Cool, and there's also `https://www.chromatic.com/library?appId=65fc89c9335768720ff8605a&branch=<branch>`

Comment (nawatts) [reply .github/pull_request_template.md:17]: Someone has made an Action to comment on the PR with a link to the preview URL.
https://github.com/dannyhw/storybook-chromatic-link-comment

If we don't want a 3rd party action, we could also add a job summary on the workflow run page.
https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#adding-a-job-summary

Comment (cahrens) [reply .github/pull_request_template.md:17]: I was thinking that there would be a way to refer to the branch name (a variable) within the PR template. But some quick Googling shows that I am wrong.

Comment (nawatts) [reply .github/pull_request_template.md:17]: Yeah, the PR template is static. But an Actions workflow _could_ update the description when the PR is opened.

**Pull request merged**